File: Ejercicio 24/ej24.py
l1 = [1,2,3,4,5]
l2 = [1,2,4,5]

n = 5


def is_permutation(lis, num):
    l3 = [i for i in range(1, num+1)]
    if l3 == lis:
        return True
    else:
        return False

print(is_permutation(l1, n))
print(is_permutation(l2, n))

File: Ejercicio 24/ej24.js
let l1 = [1,2,3,4,5];
let l2 = [1,2,4,5];

let n = 5;


function isPermutation(list, num){
    l3 = [];
    for(let i = 1; i <= num; i++){
        l3.push(i);
    }

    if (l3.length != list.length){
        return false;
    }
    
    for(let i = 0; i <= n; i++){
        if(l3[i] != list[i]){
            return false;
        }
    }
    return true;
}

console.log(isPermutation(l1, n));
console.log(isPermutation(l2, n));